djangocms_versioning/admin.py

```diff
@@ -816,6 +815,15 @@ def changelist_view(self, request, extra_context=None):
             )
         except (TypeError, ValueError):
             grouper = None
+        else:
+            if grouper is None:
```

Comment:
Is this check redundant?

Reply:
No. if the grouper is None, not because there was an error, but because no grouper exists we want to redirect to the grouper page where the user can select a valid grouper. 

If the there is an exception an error page will be shown instead.